src/site/stages/build/process-cms-exports/schemas/output/file.js

```diff
@@ -2,13 +2,8 @@ module.exports = {
   type: 'object',
   properties: {
     contentModelType: { enum: ['file'] },
-    derivative: {
-      type: 'object',
-      properties: {
-        url: { type: 'string' },
-      },
-      required: ['url'],
-    },
+    filename: { type: 'string' },
```

Comment:
Where do you see that this should be `filename`? (It sounds right, I just don't remember where it's used.)

Reply:
`filename` is used in the `media-document` transformer.